bytes/bytes.gobra

```diff
@@ -0,0 +1,33 @@
+//+gobra
```

Comment:
Can you please add a license to these files? The following would be ok:
```
// Any copyright is dedicated to the Public Domain.
// http://creativecommons.org/publicdomain/zero/1.0/
```

Reply:
Should I use public domain or the same header as in byteslice.gobra?